components.yaml

```diff
@@ -25,10 +25,10 @@ components:
     metadata: v0.9.0
   multus:
     url: https://github.com/k8snetworkplumbingwg/multus-cni
-    commit: f4c0adf54c99d7395d30a050a8d29b674b99d700
```

Comment:
Since there a re DNM commits in the PR then I am assuming this PR is still WIP, but just to understand - do you plan to pin multus to a specific commit? if so - please set `update-policy` to "static", and `metadata` to the virtual tag (the latter is not necessary but nice to have).

if not - then perhaps it's best to keep the `update-policy` at "tagged" and follow the thick plugin branch (I'm guessing [this](https://github.com/k8snetworkplumbingwg/multus-cni/tree/feature/multus-4.0))?

Reply:
that branch was a feature development branch. It is no longer being pushed to.

There isn't a tag with multus v4; that's the reason why I'm pointing to a commit. Thus, I cannot point metadata to a tag.

I'm not saying I want to point to a commit - right now, I literally do not have a choice since multus does not have a tag I can consume. 

... I still have to figure out what's wrong w/ the macvtap tests.

Reply:
Done. 

Let's see if this makes CI happy ... :crossed_fingers: